### PR TITLE
Support filler and deepgram 

### PR DESCRIPTION
**Pull request by ali-ziaei**
`release-2023-10-13` is same as `release-2023-10-05` (current release on main)
`release-2023-10-12` has filler change and deepgram support

### CODE REVIEW

File: vocode/streaming/transcriber/azure_transcriber.py [hunk 1]
```diff
@@ -74,54 +74,60 @@ def __init__(
...
-            for phrase in transcriber_config.phrases:
+            for phrase in self.transcriber_config.phrases:
                 speech_phrase_list_grammer.addPhrase(phrase)
 
         self._ended = False
         self.is_ready = False
 
     def recognized_sentence_final(self, evt):
+        start = evt.result.offset / 1e7
+        duration = evt.result.duration / 1e7
```
Comment (matthew-ceravolo) [inline vocode/streaming/transcriber/azure_transcriber.py:112]: where's the `1e7` come from?

File: vocode/streaming/models/events.py [hunk 2]
```diff
@@ -38,4 +39,17 @@ class RecordingEvent(Event, type=EventType.RECORDING):
...
-    action_output: Optional[dict] = None
\ No newline at end of file
+    action_output: Optional[dict] = None
+
+
+class FillerEvent(Event, type=EventType.PUBLISH_FILLER):
+    filler_phrase: str
+
+
+class SpokenMetaData:
```
Comment (matthew-ceravolo) [inline vocode/streaming/models/events.py:49]: nit: typically it would be `Metadata` with a lower `d`

also, not sure how useful the comment below is :)

**Review by matthew-ceravolo — APPROVED** [review]

File: vocode/streaming/transcriber/azure_transcriber.py (hunk 1 above)
Comment (ali-ziaei) [reply vocode/streaming/transcriber/azure_transcriber.py:112]: https://learn.microsoft.com/en-us/azure/ai-services/speech-service/get-speech-recognition-results?pivots=programming-language-python

unit is 100 nano seconds

File: vocode/streaming/models/events.py (hunk 2 above)
Comment (ali-ziaei) [reply vocode/streaming/models/events.py:49]: will fix and add more docstring

**Pull request merged**